llvm/lib/Target/AArch64/AArch64Subtarget.cpp

```diff
@@ -143,6 +143,7 @@ void AArch64Subtarget::initializeProperties(bool HasMinSize) {
   case CortexA78AE:
   case CortexA78C:
   case CortexR82:
+  case CortexR82AE:
```

Comment:
Can you move both of these into the block with CortexA55? It has always been in the wrong place compared to the other cpus around it and the alignments it sets.

Reply:
Thanks, done.